cmd/livepeer_cli/wizard_stats.go

```diff
@@ -29,9 +30,15 @@ func (w *wizard) stats(tips bool) {
 	fmt.Fprintf(wtr, "Eth balance: \t%s\n", w.getEthBalance())
 	wtr.Flush()
 
-	w.broadcastStats()
-	w.transcoderStats()
-	w.delegatorStats()
+	if ( strings.Compare(w.getTranscoderStatus(),"False" )  == 0){
+		w.delegatorStats()
```

Comment:
@ericxtang Is there already an endpoint for checking the transcoder status ?

Reply:
There isn't a transcoder status endpoint yet.  We would have to follow the steps in https://github.com/livepeer/go-livepeer/issues/150 and add one.  But let's wait until the 2 existing PRs get merged before we start on that.  There are some some big code changes coming in the next few days.

Reply:
what does w.getTranscoderStatus() does ?

Reply:
It gets the "onchain" transcoder status - https://github.com/livepeer/go-livepeer/blob/master/server/webserver.go#L343 

You can also take a look in the eth/client.go code to get a sense of all the calls to the blockchain.  This might be hard at the beginning, but if you refer to the eth client and the whitepaper, you can start to get a deeper understanding of the protocol.